docs/deploy.md

```diff
@@ -254,8 +245,13 @@ vagrant destroy -f a1
 vagrant destroy -f p1
 ```
 
+# Shutting Down or Deleting Your Cluster
 
-# Destroy
+The following command stops your running cluster:
+
+```
+vagrant halt
```

Comment:
vagrant halt works, but you can't bring it up again afterwards. So for now you have to destroy.